host/cluster-down.yaml

```diff
@@ -433,3 +421,14 @@ spec:
           - pass:
               when: "true"
               message: "ETCD healthz check using Kubernetes API is OK"
+    - textAnalyze:
+        checkName: ETCD API Health
+        fileName: host-collectors/run-host/curl-etcd-health-2379.txt
+        regex: ".*\"health\":\"true\"*"
+        outcomes:
+          - fail:
+              when: "false"
+              message: "ETCD status returned: unhealthy"
+          - pass:
+              when: "true"
+              message: "ETCD is health"
```

Comment:
Like the above message, I'd err on the side of slightly more words here to be unambiguous in our meaning.

Reply:
WDYT now?